### PR TITLE
Get address to lat lng

### PR DESCRIPTION
**Pull request by DennisMaas**
you can now get the address from maps when setting the marker.
the street is no longer needed and therefore deleted in the model and everywhere else.

### CODE REVIEW

File: frontend/src/components/commons/PlaceForm.js [hunk 1]
```diff
@@ -59,14 +59,26 @@ const types = [
...
+      .get(url)
+      .then((response) => response.data)
+      .then((data) => {
+        /*want to filter array results for type "sublocality_level_2" and then print formatted_adress to placeData.address*/
+        const getAddress = data.results[0].formatted_address;
+        setPlaceData({ ...placeData, address: getAddress, lat: lat, lng: lng });
+      })
+      .catch(console.error);
+  };
+  console.log(placeData.address);
```
Comment (christophersiem) [inline frontend/src/components/commons/PlaceForm.js:81]: remove console.log if not needed anymore

File: frontend/src/components/googleMaps/FetchAddress.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export default function FetchAddress() {
+  const key = process.env['REACT_APP_MAPS_API_KEY'];
+  const lat = placeData.lat;
+  const lng = placeData.lng;
+  const url = `https://maps.googleapis.com/maps/api/geocode/json?latlng=${lat},${lng}&key=${key}`;
+
+  fetch(url)
+    .then((response) => response.json())
+    .then((data) => {
+      console.log(data);
```
Comment (christophersiem) [inline frontend/src/components/googleMaps/FetchAddress.js:10]: remove console.log if not needed anymore

**Review by christophersiem — COMMENTED** [review]
looking good 🏘

File: frontend/src/components/commons/PlaceForm.js (hunk 1 above)
Comment (DennisMaas) [inline frontend/src/components/commons/PlaceForm.js:81]:
```suggestion
```

File: frontend/src/components/googleMaps/FetchAddress.js (hunk 2 above)
Comment (DennisMaas) [inline frontend/src/components/googleMaps/FetchAddress.js:10]:
```suggestion

```

**Pull request merged**